### PR TITLE
fix(Alert): Prevent JS test loop for line defaults

### PR DESCRIPTION
**Pull request by codecaaron**
### Overview

* Removes initialization lines and let Truncation escape out if not configured.

### PR Checklist

- [ ] Related to designs:
- [ ] Related to JIRA ticket: ABC-123
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Review by JoshuaKGoldberg — APPROVED** [review]

File: packages/gamut/src/Alert/index.tsx [hunk 1]
```diff
@@ -77,8 +77,8 @@ export const Alert: React.FC<AlertProps> = ({
             justify="spaceBetween"
           >
             <Truncate
-              expanded={isExpanded}
-              lines={lines}
+              expanded={isExpanded || Boolean(lines)}
```
Comment (JoshuaKGoldberg) [inline packages/gamut/src/Alert/index.tsx:80]: Think this is unit testable?

Comment (codecaaron) [reply packages/gamut/src/Alert/index.tsx]: Yeah, fixing the tests for this <3

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.4.15-alpha.5741f0.0<br/>@codecademy/gamut-labs@7.16.3-alpha.5741f0.0<br/>@codecademy/gamut@20.0.2-alpha.5741f0.0<br/>@codecademy/styleguide@22.0.2-alpha.5741f0.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60131efc1768fe5ea25925c4--gamut-preview.netlify.app>https://60131efc1768fe5ea25925c4--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60131efc1768fe5ea25925c4>Deploy Logs</a></p>

**Pull request merged**